cmake/Modules/LLVMVersion.cmake

```diff
@@ -0,0 +1,15 @@
+# The LLVM Version number information
+
+if(NOT DEFINED LLVM_VERSION_MAJOR)
```

Comment:
I think this is worthwhile as its own change, and in particular we could probably derive `_LIBCPP_VERSION` from it, which would simplify the job of the release manager.

Reply:
This is now https://github.com/llvm/llvm-project/pull/84641